packages/client/tests/functional/row-select-update/prisma/_schema.ts

```diff
@@ -0,0 +1,21 @@
+import { idForProvider, idTypeForProvider } from '../../_utils/idForProvider'
+import testMatrix from '../_matrix'
+
+export default testMatrix.setupSchema(({ provider, providerFeatures }) => {
+  return /* Prisma */ `
+    generator client {
+      provider = "prisma-client-js"
+      previewFeatures = [${providerFeatures}]
+    }
+    
+    datasource db {
+      provider = "${provider}"
+      url      = env("DATABASE_URI_${provider}")
+    }
+    
+    model Resource {
+      id       ${idForProvider(provider)}
+      occStamp Int @default(0) @unique
```

Comment:
This is to easily expose the behavior on single `update`s, which require unique keys in the where condition. This works well because we are only doing occ on one item.